.github/workflows/build.yml

```diff
@@ -38,7 +38,9 @@ jobs:
       - name: Store CI fixes in upstream artifact
         run: |
           mkdir -p upstream
-          git format-patch --stdout test_base > upstream/ci_fixes.patch
+          if git format-patch --stdout test_base > ci_fixes.patch; then
+            cp ci_fixes.patch upstream/
+          fi
```

Comment:
Why does it "cp" instead of "mv"?

Reply:
No particular reason. Except just using `mv` instead of `cp` would look unbalanced to me, and I would be compelled to add `else rm ci_fixes.patch`

Reply:
OK. No big deal, as container is ephemeral.